Representable/representable
# user flow: new thanks page

**changes**
- removed the thanks page (template, view, url)
- submitting an entry now redirects to /submission/thanks/map_id or /submission/thanks/org-slug/drive-slug/map_id
- fixed links to submission page to work with new link format
- on submission thanks url, displays a modal with link to submission, link to drive map, and feedback link

**to test**
- try submitting a community through drive link
- check that you are redirected to submission page with thanks modal, and that thanks modal has working drive link
- try submitting a community outside of a drive
- check that you are redirected to submission page with thanks modal, and that thanks modal has no drive link visible
- check links around the site to individual submissions, or to the drive page

**screenshots**
with drive
![image](https://user-images.githubusercontent.com/41943646/95257751-cb58b300-07f2-11eb-82ec-d64701b3916d.png)

without drive 
![image](https://user-images.githubusercontent.com/41943646/95257918-04912300-07f3-11eb-8453-7b016093f54a.png)
